cms/djangoapps/api/v1/serializers/discussion_settings.py

```diff
@@ -0,0 +1,43 @@
+""" Discussion settings serializers. """
+
+
+from rest_framework import serializers
+
+from models.settings.course_metadata import CourseMetadata
+from common.lib.xmodule.xmodule.course_module import validate_blackout_datetimes
+
+
+def blackout_date_range_validator(blackout_dates):
+    """
+    Given two datetime object, checks if 2nd date/datetime is larger than
+    1st date/datetime. returns ``value`` as is if valid,
+    else raises ``serializers.ValidationError``.
```

Comment:
I don't think it's validating this anymore. I am able to enter two a starting data that is later than the ending data and it works.